kaggle/RANZCR/4th_place_solution/data/seg_data.py

```diff
@@ -0,0 +1,113 @@
+import ast
+
+import cv2
```

Comment:
Is opencv installed by default? We may need so provide some install instructions